### ISSUE TITLE
Adoption/Support?

### ISSUE THREAD

**Issue by JessicaMulein** [csharptest/CSharpTest.Net.Collections#21]
I'm interested in potentially adopting this as part of BrightChain if you're no longer interested in maintaining it. But what lead you to abandon it? I'd love to keep your help, but would be happy to try and help you out as your code is ideal for what I'm doing. Please let me know if I can help monetarily or with labor.

**Comment by NightOwl888** [comment]
It seems that @csharptest hasn't been active on GitHub in some time and apparently, he shelved this project and [no longer even works with .NET or Windows](http://csharptest.net/1299/hello-is-anyone-there/index.html).

I am just an interested party who tried to get this library updated to use .NET Standard, but ended up pulling out `LurchTable` and customizing it for my own needs.

I noticed that `BPlusTree` is also a dependency of Umbraco, and there is a list of [some of the bugs in `BPlusTree`](https://github.com/umbraco/Umbraco-CMS/issues/8447) in their GitHub issues list.

Also, I noticed that BrightChain is covered under GNU, which is not a license that allows linking with another license. I haven't looked in depth at the architecture of BrightChain, but I am sure others would appreciate if the license remained open, allowing others to link to the software without requiring a GNU/GPL license.

If there are enough interested parties willing to provide some long term funding for support as well as either funding or some labor to help migrate, patch, and rename the library, I would be willing to be the new maintainer of the library. This doesn't really fit in with the purpose of [J2N](https://github.com/NightOwl888/J2N) (where I put `LurchTable`), but I have quite a few build scripts already templated for versioning/build/test/deploy to quickly get it up on GitHub and NuGet as a new library. I also found some bugs in the tests that rely on the implementation of .NET Framework that I know how to fix and know about some of the issues that are preventing these collections from being compatible with Xamarin iOS.

Contact me at the email address on my GitHub profile if you are interested.
